inference-engine/cmake/features.cmake

```diff
@@ -77,6 +77,8 @@ ie_dependent_option (ENABLE_BEH_TESTS "tests oriented to check inference engine
 
 ie_dependent_option (ENABLE_FUNCTIONAL_TESTS "functional tests" ON "ENABLE_TESTS" OFF)
 
+ie_dependent_option (ENABLE_CONFORMANCE_TESTS "conformance tests" ON "ENABLE_TESTS" OFF)
```

Comment:
If we decided not to add additional target, why we need this make definition?

Reply:
I don't know what you decided :) I have added additional target for conformance. As minimum, there are 2 targets - runner and subgraph dumper